lib/test_prof/factory_default.rb

```diff
@@ -168,6 +168,10 @@ def set_factory_default(name, obj, preserve_traits: FactoryDefault.config.preser
         )
       end
 
+      def get_factory_default(name, *args)
+        FactoryDefault.get(name, args, args.extract_options!, false)
```

Comment:
Why can't we use `*args, **kwargs` instead of `#extract_options!`? No need to support legacy Hash-to-kwarg conversion, it's a new API method.